backend/logic/deploy/dns.js

```diff
@@ -0,0 +1,83 @@
+const setCloudflareRecords = require('../../utils/dns/cloudflare')
+const setCloudDNSRecords = require('../../utils/dns/clouddns')
+const setRoute53Records = require('../../utils/dns/route53')
+const { getLogger } = require('../../utils/logger')
+
+const log = getLogger('logic.deploy.dns')
+
+/**
+ * Copy all needed files for a shop deployment to the public build directory and
```

Comment:
This does not seem like the correct description of what the method does? copy/pasta?